docker/start_container.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+echo "Starting container..."
+if [ -z $(docker ps --filter=name='citrixitm_tf_dev_container' -q) ]
+then
+    if [ -z $(docker ps --filter=name='citrixitm_tf_dev_container' -a -q) ]
+    then
+        echo "The citrixitm_tf_dev_container container doesn't exist. Did you mean \"make docker-run\"?"
```

Comment:
you could use single quotes here as well to be consistent with line 13 - `echo 'The ...'

Reply:
Fixed.